### PR TITLE
Update lerna to v4

### PR DESCRIPTION
**Pull request by millerds**
Clears up all but 2 vulnerabilities resulting from lerna dependancy

### CODE REVIEW

**Review by akrantz — APPROVED** [review]

File: packages/eslint-config-office-addins/package.json [hunk 1]
```diff
@@ -16,6 +16,7 @@
   },
   "devDependencies": {
     "@types/node": "^12.12.34",
+    "@types/es6-promise": "^3.3.0",
```
Comment (akrantz) [inline packages/eslint-config-office-addins/package.json:19]: Is this related to switching lerna versions? Seems like this PR is doing more than one thing.

Comment (millerds) [reply packages/eslint-config-office-addins/package.json:19]: check your teams :-).  For some reason updating lerna versions exposed these missing pieces of a few package.json files causing the builds to break (somehow missed this when I build locally the first time).  That why the CI loop didn't pass the first time.

**Review by akrantz — APPROVED** [review]
